lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
And to be more precise, `ActiveSupport.on_load(:action_controller)` is not triggered when `ActionController` is loaded, but when `ActionController::Base` is.